haikudepotserver-core/src/main/resources/userusageconditions/UUC2019V01.md

```diff
@@ -61,7 +61,7 @@ The HDS system is able to store the following non-identifying information about
 * The user's preferred natural language; examples would be Polish, Japanese or Spanish.  This information is used, where possible, to display the HDS user interface in the language best suited to the user.
 * A one-way hashed password used as part of the authentication mechanism.
 * The date and time of the user's last authentication.  This can be used to determine if a user has been inactive for a period of time.
-* The date and time at which the user agreed to a specific version of the user usage conditions.  As part of the agreement, the user may need to confirm that they are as old or older than some minimum age.  The minimum age forms part of the user usage conditions.
```

Comment:
The conditions file `UUC2019V01.md` is not able to be edited once it is deployed; it is a fixed version of the wording.  Instead a new version of the conditions needs to be loaded in.  I would prefer to defer this until significant change is being considered and then at that time it would be possible to make these tweaks.  Are you able to remove this change and update the pull-request.

Reply:
Hi @aplgithub  you meant @humdingerb not me @humdinger

Reply:
Ah sorry thanks!